python-lib/dku_model_fairness_report/fairness_metric_report.py

```diff
@@ -69,15 +77,26 @@ def __init__(self, y_true, y_pred, sensitive_feature_values, advantageous_outcom
         self.sample_weight = sample_weight  # 1D array
         self._check()
 
+        self.label_list = self.get_label_list()
+
     def _check(self):
         possible_outcomes = set(self.y_true.unique()).union(set(self.y_pred.unique()))
         if self.advantageous_outcome is not None and self.advantageous_outcome not in possible_outcomes:
-            raise ValueError('The chosen advantageous outcome, "{}", does not exist in either y_true or y_pred.'.format(self.advantageous_outcome))
+            raise ValueError('The chosen positive outcome, "{}", does not exist in either y_true or y_pred.'.format(self.advantageous_outcome))
+
+        if len(possible_outcomes) != 2:
+            raise ValueError('Only support binary classification, found {} possible values.'.format(len(possible_outcomes)))
 
     def _check_reference_group(self, reference_group, summary):
         if reference_group not in summary.get(DkuFairnessConstants.BY_GROUP).keys():
             raise ValueError('The chosen reference group "{0}" does not exist in the input metric summary.'.format(reference_group))
 
+    def get_label_list(self):
+        label_list = [x for x in np.unique(self.y_true) if x != self.advantageous_outcome]
```

Comment:
You might be willing to explain that you keed unique values and put the advantageous outcome at the end. Does it always belong to the self.y_true ?

Reply:
Yep there is a check for that